docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -118,7 +120,7 @@ When you query Twitter API, you will get all tweets matching the query without c
 
 For example, let's get all tweets talking about GSB and Stanford since December 1, 2022 until today.
 
-```python
+```bash linenums="1" hl_lines="4-4" title="python"
```

Comment:
same here

Reply:
I think you're looking at outdated file but I changed all python to py